menu_group/build.gradle

```
@@ -0,0 +1,3 @@
+dependencies {
+    implementation project(path: ':menu')
```

```suggestion
    implementation project(':menu')
```

이렇게도 적을 수 있습니다! 바꿔달라는 건 아니고 정보 전달 관점에서 커멘트 남겼습니다 ㅎㅎ